server/src/integration/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLampIT.java

```diff
@@ -36,7 +36,7 @@
     private final String envSpace = System.getenv(CF_ENVIRONMENT_SPACE);
 
     public CloudFoundryLampIT() {
-        super(new CloudFoundryPlugin());
+        super(new Plugin());
```

Comment:
I forgot to mention that you should have excluded the CloudFoundry Plugin and lifecylce class from this suggestion. As well as the corresponting tests. Sorry for that "inconvenience"